### PR TITLE
Po dupes

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: translate/storage/html.py [hunk 1]
```diff
@@ -326,6 +328,7 @@ def endblock(self):
         self.currentcomment = ""
         self.currenttag = None
         self.currentpos = -1
+        self.currentoffset = 0
```
Comment (dwaynebailey) [inline translate/storage/html.py]: I'm guessing that you might actually want this to be -1 i.e. not defined.

File: translate/storage/html.py [hunk 2]
```diff
@@ -313,6 +314,7 @@ def startblock(self, tag, attrs=None):
         self.currentcomment = ""
         self.currenttag = tag
         self.currentpos = self.getpos()[0]
+        self.currentoffset = self.getpos()[1]
```
Comment (dwaynebailey) [inline translate/storage/html.py]: `self.currentpos, self.currentoffset = self.getpose()[:1]` might be neater, up to you.

File: translate/storage/html.py [hunk 3]
```diff
@@ -176,6 +176,7 @@ def __init__(self, includeuntaggeddata=None, inputfile=None,
         self.currentcomment = u""
         self.currenttag = None
         self.currentpos = -1
+        self.currentoffset = 0
```
Comment (dwaynebailey) [inline translate/storage/html.py]: Ditto here `-1` as in not defined yet.

**Comment by dwaynebailey** [comment]
"Adjustments to match the implementation" ? Needs a better commit message.

**Comment by dwaynebailey** [comment]
I don't see any tests adjusted for html or txt? If the tests are missing I'd recommend adding them now.

**Comment by unho** [comment]
Addressed all raised concerns.

**Pull request merged**